Gemfile

```
@@ -53,6 +53,11 @@ gem 'image_processing', '~> 1.2'
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rspec-rails', '~> 6.1.0'
+  %w[rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
+    gem lib, git: "https://github.com/rspec/#{lib}.git", branch: 'main'
+  end
```

```suggestion
  gem 'rspec-rails'
```

だけで十分だと思うのですが、わざわざ遠回りな記述をしている理由が知りたいです〜。
（ネットの情報等を参考にしたのであればURLを教えてください）

---

削除しました。

正直あまり、自分でもどうしてこうしたのか覚えていないですね。specがうまく動作しない時があって、何か必要なものが入っていないのかなと試行錯誤の中で残ったコードかもしれません。

https://rspec.info/documentation/6.1/rspec-rails/

---

情報ありがとうございます！なるほど、ここを参考にしたんですね〜。
ただ、

> if you want or require the latest unstable features.
>
> Or, run against the main branch

とあるので、普通は`gem 'rspec-rails', '~> 6.1.0'`だけでいいんだけど、もし・・・という体（てい）で書かれているのでeachで回す処理は基本的に不要ですね